packages/datepickers/src/styled/StyledRangeCalendar.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { StyledDatepicker } from './StyledDatepicker';
+import { DEFAULT_THEME, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'datepickers.range_calendar';
+
+export const StyledRangeCalendar = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID
+})`
+  display: flex;
+  overflow: auto;
+
+  ${StyledDatepicker} {
+    padding: 0;
+  }
+
+  ${props =>
+    props.theme.rtl
+      ? `${/* sc-selector */ StyledDatepicker}:last-of-type {margin-right: ${props.theme.space
+          .base * 5}px}`
+      : `${/* sc-selector */ StyledDatepicker}:first-of-type {margin-right: ${props.theme.space
+          .base * 5}px}`}
```

Comment:
Can this move under the `${StyledDatepicker}` block above as `&:last-of-type` and `&:first-of-type`?

Reply:
I've used a construct like this in other packages:

```js
&:{props => props.theme.rtl ? 'first' : 'last'}-of-type: { margin-right: ...; }
```

Reply:
Oops, please note that Abstract specifies 16px space between calendars in compact mode.

Reply:
This value was purposefully increased during development with the designer. Let's revisit this as a separate task if this is something that we want to update.

Reply:
> I've used a construct like this in other packages

Prettier starts to format values into invalid CSS with different combinations. This was the most readable option with formatting for me.

I have raised it into the parent `${StyledDatepicker}` block though.